### DISCUSSION TITLE
Add a way for us to change the key binds

### DISCUSSION THREAD

**Discussion by Shamus4** [zippity21/Stardew_Valley_Regression_Mod#66] [Ideas]
potential to change key binds as some of these are already used by other mods/ basic functions of my computer

**Comment by zippity21** [comment]
> potential to change key binds as some of these are already used by other mods/ basic functions of my computer

Great idea, I've always worried about overlap. I don't know how to implement it, but I'll put it on the list of enhancements.

**Comment by Gooberology** [comment]
I think i can try a crude implementation of it via the config file. Wont be as good as in game keybinds, but its better than nothing,

**Reply by zippity21** [reply]
Someone, at some point, recommended using [CMGM](https://www.nexusmods.com/stardewvalley/mods/5098?tab=posts) to control things like this. I haven't investigated the approach yet, but it seems like a common thing that mods use.
